nimbus.fml.yaml

```diff
@@ -0,0 +1,30 @@
+channels:
+  - debug
+  - nightly
+  - beta
+  - release
+features:
+  tabs:
+    description: Nimbus feature name intended to control the multiple tabs feature in the app.
+    variables:
+      is_multi_tab:
```

Comment:
Nit: use snake case here.

Also, naming: I like that it's called `tabs`; but is it really `is-multi-tab` or is it just `is-enabled` (I'm confused by a tabs feature without multi-tabs).

Reply:
This feature is already in production, in this pr it was just migrated to FML. In my opinion we should create another ticket to address this comment and not add regressions

Reply:
> create another ticket to address this comment and not add regressions
Agreed. Please file a migration ticket which should be done after once experiments and rollouts on `is_multi_tab` are complete.

Reply:
https://github.com/mozilla-mobile/focus-android/issues/6901